src/components/customer-resource-show.js

```diff
@@ -143,3 +156,6 @@ CustomerResourceShow.propTypes = {
   toggleRequest: PropTypes.object.isRequired,
   toggleSelected: PropTypes.func.isRequired
 };
+CustomerResourceShow.contextTypes = {
+  intl: PropTypes.object
```

Comment:
it looks like we're getting this from props in some places, context in others.  we should probably pick one or the other, but i dunno enough about the domain to know which is correct

Reply:
Could use help here as well, it seems contextType has benefits - but need to understand if this is proper usage

Reply:
So researching more on propType vs contextType https://javascriptplayground.com/blog/2017/02/context-in-reactjs-applications/
Updated coverage-dates component to rec'v intl as context (instead of property). Is there somewhere else that looks inconsistent? or is there a best practice to use property instead of context?